install/index.markdown

```diff
@@ -27,7 +27,7 @@ Choose your ROS distribution below:
 
 *Note for Ubuntu 13.04 32 bit users*: There is a bug with GCC 4.7 on Ubuntu 13.04 32bit with Eigen 3.1.2. It's not likely to be fixed, so upgrade/downgrade your system to 13.04 64 bit resp. 12.04.
 
-### Ubuntu Installation: Debian Packages for MoveIt!
+### Install Debian Packages
```

Comment:
There are no *Debian* packages for MoveIt!. That implies they are available in debian.
`ros-indigo-moveit-*` are deb packages built by osrf